azure/aks/infrastructure/variables.tf

```diff
@@ -0,0 +1,77 @@
+variable "ssh_public_key" {
+  description = "Path to SSH public key passed as authorized key in kubernetes node group"
+  type        = string
+  default     = "~/.ssh/id_rsa.pub"
```

Comment:
Using this default is a safe solution? :thinking:

Reply:
Will move it to a variable and add a note in readme.md

Reply:
Removed the default and added the info about it in the README.md file